src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "GROUPS_TABLE")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
```

Comment:
Ej ale mi kazałeś usunąć Gettery i Settery a sam masz :P

Reply:
Tam chodziło mi o kolejność adnotacji, a co do getterów i setterów to czekam na odp Huberta, bo różne informacje można znaleźć. Widizałem encje bez adnotacji @Getter jak i  informacje że gettery i settery są obowiązkowe.

Reply:
Ok, myślę, że to nie ma znaczenia na razie, bo ich nie używamy. Nie jest to też taki błąd czy coś, żeby nie puszczać pull requesta. Spróbuj usunąć Group Dto i daje aproval🙂

Reply:
To może dajmy te gettery wszędzie?